include/seqan/sequence/string_set_owner.h

```diff
@@ -141,6 +141,21 @@ class StringSet<TString, Owner<Default> >
 // ============================================================================
 
 // --------------------------------------------------------------------------
+// Function append()
+// --------------------------------------------------------------------------
+
+// better solution if both stringsets are Owner<Default>
+template <typename TString, typename TString2, typename TExpand >
+inline void append(StringSet<TString, Owner<Default> > & me,
+                   StringSet<TString2, Owner<Default> > const & obj,
+                   Tag<TExpand>)
+{
+    // we rather invalidate limits here to allow to do modify appended strings:
+    me.limitsValid = false;
+    append(me.strings, obj.strings, Tag<TExpand>());
+}
+
+// --------------------------------------------------------------------------
```

Comment:
To be honest, I am not sure this is completely the same, as this intricacies of `AppendString_` and the entire arrayCopy* is not very straight-forward... I would prefer not do any major changes there now, as we want to re-evaluate the whole container situation anyway, right?